base/c.jl

```diff
@@ -49,7 +50,7 @@ Equivalent to the native `wchar_t` c-type ([`Int32`](@ref)).
 """
 Cwchar_t
 
-if !is_windows()
+if ccall(:jl_get_UNAME, Any, ()) !== :NT
```

Comment:
Also the first `ccall` works fine, it's just this second one that's getting upset